trainee_template/derive_utilities.amlg

```diff
@@ -76,11 +76,16 @@
 							(first (get react_output "action_values"))
 						)
 
-						(if output_influential_cases
-							(accum (assoc influential_cases (get react_output "influential_cases") ))
-						)
-						(if output_cap
-							(accum (assoc local_class_probabilities_map (get react_output "categorical_action_probabilities") ))
+						(if output_details
+							(assign (assoc
+								case_detail_values_map
+									(map
```

Comment:
def add a comment explaining what this is doing

Reply:
will do.